src/ui/ship_building/ship_building_view.gd

```diff
@@ -12,7 +12,7 @@ var bgm_music
 
 func _ready():
 	TickManager.tick_changed.connect(_update_star_particles)
-	EventManager.building_finished.connect(tutorial_tracker)
+	EventManager.building_finished_signal.connect(tutorial_tracker)
```

Comment:
Do we need to call it `building_finished_signal`? We don't call int variables `housing_value_int`, the fact we're connecting it to a method here infers the fact that it's a signal and we shouldn't have any other vars that have the same name as this signal.

Reply:
Yeah this is silly, I changed it back